LetsDoIt.CustomValueTypes/src/LetsDoIt.CustomValueTypes/Email.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Net.Mail;
+using System.Text.Json.Serialization;
+
+namespace LetsDoIt.CustomValueTypes
+{
+
+    [JsonConverter(typeof(EmailJsonConverter))]
+    public readonly struct Email
+    {
+        private readonly string _value;
+
+        public Email(string value)
+        {
+            if (value == null)
```

Comment:
We already check if the email is null in IsValid method. So we can remove this if condition.

Reply:
Done.